src/app.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, delay=0):
             CsvPatternLogger(),
         ]
         self.visualizer = Visualizer("Press 'q' to quit")
-        self.tts_patterns = TtsPatterns()
+        self.tts_patterns = TtsPatterns(quit)
```

Comment:
what is `quit`? this variable is not defined anywhere and not used anywhere

Reply:
forgot to delete after an idea, thanks for the hint.